projects/observability/src/shared/components/topology/d3/d3-topology.ts

```diff
@@ -410,6 +417,8 @@ export class D3Topology implements Topology {
       this.neighborhoodFinder.singleNodeNeighborhood(node.userNode)
     );
 
+    this.checkAndHandleGroupNodeClick(node);
```

Comment:
Can you separate the entire new code with an input property? This way, we can protect the existing app flow and avoid bugs.

Reply:
Done